### PR TITLE
integrated login with latest master

### PR DESCRIPTION
**Pull request by Initech9**
I will always use your latest master when updating my branches. I kept the .env.example intact and I left the welcome.blade.php intact. It says Laravel Management System now too.

mergedmaster

### CODE REVIEW

**Comment by Initech9** [comment]
I made a small banner header in the public/images folder... we can change the colors if you'd like
volunteer

File: resources/views/members/show.blade.php [hunk 1]
```diff
@@ -95,10 +86,4 @@
             @endforeach
         </tbody>
     </table>
-    <h3>Interests</h3>
-    <ul>
-        @foreach ($member->interests as $interest)
-            <li>{{ $interest->interest }}</li>
-        @endforeach
-    </ul>
```
Comment (stephenhartman) [inline resources/views/members/show.blade.php]: This block should stay, it displays the member interests

File: resources/views/members/show.blade.php [hunk 2]
```diff
@@ -48,16 +48,7 @@
                 @endif
                 <td>{{ ucfirst($member->approval_status) }}</td>
                 <td>
-                    <div class="row">
```
Comment (stephenhartman) [inline resources/views/members/show.blade.php]: This block should stay as well, it organizes the delete and edit buttons.  Not functionally, it is a mockup, but a good starting point that should stay.

File: resources/views/welcome.blade.php [hunk 3]
```diff
@@ -13,8 +13,12 @@
         @endif
 
         <div class="content">
+		<img src="{{url('/images/volunteer.jpg')}}" alt="Image"/>
             <div class="title m-b-md">
-                Laravel
+
```
Comment (stephenhartman) [inline resources/views/welcome.blade.php]: We shouldn't leave whitespace floating around like this, clean code is readable code.

File: resources/views/layout.blade.php [hunk 4]
```diff
@@ -4,13 +4,35 @@
...
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>Laravel</title>
+        <title>Laravel Volunteer System</title>
         <!-- Fonts -->
         <link href="https://fonts.googleapis.com/css?family=Raleway:100,600" rel="stylesheet" type="text/css">
         <link rel="stylesheet" href="{{ mix('/css/app.css') }}">
     </head>
     <body>
-        @include('_header')
```
Comment (stephenhartman) [inline resources/views/layout.blade.php]: Sorry, we included a header yesterday, so all the code below is unnecessary with this include

File: resources/views/layout.blade.php [hunk 5]
```diff
@@ -4,13 +4,35 @@
...
+                            </li>
+                        </ul>
+                        <form class="form-inline navform my-2 my-lg-0">
+                            <input class="form-control mr-sm-2" type="text" placeholder="Search">
+                            <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+                        </form>
+                    </div>
+                </nav>
+            </header>
+        </div>
```
Comment (stephenhartman) [inline resources/views/layout.blade.php]: down to here is in the _header partial

File: routes/web.php [hunk 6]
```diff
@@ -26,3 +32,21 @@
...
+  Route::post('/login', 'Auth\AdminLoginController@login')->name('admin.login.submit');
+  Route::get('/', 'AdminController@index')->name('admin.dashboard');
+  Route::get('/logout', 'Auth\AdminLoginController@logout')->name('admin.logout');
+
+  // Password reset routes
+  Route::post('/password/email', 'Auth\AdminForgotPasswordController@sendResetLinkEmail')->name('admin.password.email');
+  Route::get('/password/reset', 'Auth\AdminForgotPasswordController@showLinkRequestForm')->name('admin.password.request');
+  Route::post('/password/reset', 'Auth\AdminResetPasswordController@reset');
+  Route::get('/password/reset/{token}', 'Auth\AdminResetPasswordController@showResetForm')->name('admin.password.reset');
+});
```
Comment (stephenhartman) [inline routes/web.php]: Should there be a register route?

File: routes/web.php [hunk 7]
```diff
@@ -26,3 +32,21 @@
...
+
+
+Route::get('/home', 'HomeController@index');
+Route::get('/users/logout', 'Auth\LoginController@userLogout')->name('user.logout');
+
+Route::prefix('admin')->group(function() {
+  Route::get('/login', 'Auth\AdminLoginController@showLoginForm')->name('admin.login');
+  Route::post('/login', 'Auth\AdminLoginController@login')->name('admin.login.submit');
+  Route::get('/', 'AdminController@index')->name('admin.dashboard');
+  Route::get('/logout', 'Auth\AdminLoginController@logout')->name('admin.logout');
```
Comment (stephenhartman) [inline routes/web.php]: There is no Auth\AdminLoginController

File: routes/web.php [hunk 8]
```diff
@@ -26,3 +32,21 @@
...
+
+
+
+Route::get('/home', 'HomeController@index');
+Route::get('/users/logout', 'Auth\LoginController@userLogout')->name('user.logout');
+
+Route::prefix('admin')->group(function() {
+  Route::get('/login', 'Auth\AdminLoginController@showLoginForm')->name('admin.login');
+  Route::post('/login', 'Auth\AdminLoginController@login')->name('admin.login.submit');
+  Route::get('/', 'AdminController@index')->name('admin.dashboard');
```
Comment (stephenhartman) [inline routes/web.php]: I don't see an AdminController

File: routes/web.php [hunk 9]
```diff
@@ -26,3 +32,21 @@
 Route::resource('interests', 'InterestsController');
 
 Route::resource('volunteer_centers', 'VolunteerCenterController');
+
+
+
+Route::get('/home', 'HomeController@index');
+Route::get('/users/logout', 'Auth\LoginController@userLogout')->name('user.logout');
```
Comment (stephenhartman) [inline routes/web.php]: I do see the LoginController in Auth, along with RegisterController, ForgotPasswordController, and ResetPasswordController

File: routes/web.php [hunk 10]
```diff
@@ -26,3 +32,21 @@
...
+  Route::get('/login', 'Auth\AdminLoginController@showLoginForm')->name('admin.login');
+  Route::post('/login', 'Auth\AdminLoginController@login')->name('admin.login.submit');
+  Route::get('/', 'AdminController@index')->name('admin.dashboard');
+  Route::get('/logout', 'Auth\AdminLoginController@logout')->name('admin.logout');
+
+  // Password reset routes
+  Route::post('/password/email', 'Auth\AdminForgotPasswordController@sendResetLinkEmail')->name('admin.password.email');
+  Route::get('/password/reset', 'Auth\AdminForgotPasswordController@showLinkRequestForm')->name('admin.password.request');
+  Route::post('/password/reset', 'Auth\AdminResetPasswordController@reset');
+  Route::get('/password/reset/{token}', 'Auth\AdminResetPasswordController@showResetForm')->name('admin.password.reset');
```
Comment (stephenhartman) [inline routes/web.php]: I don't see where these Auth\Admin*Controllers are coming from.

**Comment by stephenhartman** [comment]
I tweaked a few things, and made the layout more consistent across the views.  We can improve it later.

**Pull request merged**